src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -0,0 +1,177 @@
+#![cfg(feature = "integration_tests")]
+mod test_utils;
+
+use bytes::Bytes;
+use kafka::{
+    config::ConsumerConfig,
+    test_utils::topic_scanner::KafkaTopicScanner,
+};
+use plugin_work_queue::test_utils::scan_for_plugin_message_in_pwq;
+use rust_proto::graplinc::grapl::{
+    api::{
+        event_source::v1beta1::CreateEventSourceRequest,
+        graph::v1beta1::GraphDescription,
+        pipeline_ingress::v1beta1::PublishRawLogRequest,
+        plugin_registry::v1beta1::{
+            DeployPluginRequest,
+            PluginMetadata,
+            PluginType,
+        },
+    },
+    pipeline::v1beta1::RawLog,
+};
+use test_context::test_context;
+use test_utils::context::E2eTestContext;
+use uuid::Uuid;
+
+use crate::test_utils::predicates::{
+    events_36lines_merged_graph_predicate,
+    events_36lines_node_identity_predicate,
+};
+
+mod test_fixtures {
+    use super::*;
+
+    /// Send 1 line (well, event) at a time
```

Comment:
Any particular reason we're doing this instead of just sending up the whole file?

Reply:
There was a very, very long conversation about this in #plugin-system starting with https://grapl-internal.slack.com/archives/C03JNJ57A9G/p1659408013217299 up until like noon today

Reply:
oh boy...